README.md

```diff
@@ -1,4 +1,36 @@
-[![Build Status](https://travis-ci.org/datahq/data.js.svg?branch=master)](https://travis-ci.org/datahq/data.js)
 ## Data.js
+<p align="center">
+  <a href="https://datahub.io/">
+    <img alt="datahub" src="http://datahub.io/static/img/logo-cube.png" width="146">
+  </a>
+</p>
```

Comment:
We can leave this for now but it is sort of optional since this tool is independent of datahub (we probably want its own logo ...)